lib/wit/main.py

```diff
@@ -108,6 +111,37 @@ def main() -> None:
         raise WitBug(e)
 
 
+def foreach(ws, args):
+    packages = sorted(ws.lock.packages, key=lambda x: str.lower(x.name))
```

Comment:
```suggestion
    packages = sorted(ws.lock.packages, key=lambda x: x.name.lower())
```

I think it's a bit more idiomatic in Python to call this in method form rather than function form. See https://docs.python.org/3.5/library/stdtypes.html?highlight=lower#str.lower